recipes/r-ftrading/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set version = '3042.79' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ftrading
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fTrading_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fTrading/fTrading_{{ version }}.tar.gz
+  sha256: 3e99963eb59891b14a1531f8b30692c59ae0e2a97a6d63c3ad3b37772d79ce76
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-fbasics
+    - r-timedate
+    - r-timeseries
+  run:
+    - r-base
+    - r-fbasics
+    - r-timedate
+    - r-timeseries
+
+test:
+  commands:
+    - $R -e "library('fTrading')"           # [not win]
+    - "\"%R%\" -e \"library('fTrading')\""  # [win]
+
+about:
+  home: http://www.rmetrics.org
+  license: GPL-2.0-only
+  summary: A collection of functions for trading and rebalancing financial instruments. It implements
+    various technical indicators to analyse time series such as moving averages or stochastic
+    oscillators.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

There is any additional copyright notice that we should explicitly bundle:

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - inst/COPYRIGHT.html
```